net.go

```diff
@@ -0,0 +1,27 @@
+package sqrl
+
+import (
+	"fmt"
+	"log"
+	"os/exec"
+	"strings"
+
+	"github.com/shirou/gopsutil/net"
+)
+
+func netInfo() {
+	a, _ := net.Interfaces()
+
+	fmt.Println()
+	fmt.Println("\t\t\tNetwork Info")
+
+	netSpeed, err := exec.Command("ethtool", "eno16777984").Output()
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Instead of doing a 

`log.Fatal(err)`

Consider having your netInfo() return an error that the caller can then inspect and see what happened.